command/compile.go

```diff
@@ -156,6 +158,9 @@ func registerCompile(app *kingpin.Application) {
 	cmd.Flag("secrets", "secret parameters").
 		StringMapVar(&c.Secrets)
 
+	cmd.Flag("volumes", "drone runner volumes").
```

Comment:
Add some more doc on what's this volume and how it is used?

Reply:
added a comment for reference